File: backend/voting-app/src/main/java/com/tu/votingapp/dto/response/DocumentResponseDTO.java
package com.tu.votingapp.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DocumentResponseDTO {
    private Long id;
    private String number;
    private LocalDate validFrom;
    private LocalDate validTo;
    private String issuer;
    private Integer gender;
    private LocalDate dateOfBirth;
    private String permanentAddress;
    private Long userId;
}

File: frontend/android/app/src/main/java/com/example/android/dto/response/DocumentResponseDTO.kt
package com.example.android.dto.response

import kotlinx.serialization.Serializable

@Serializable
data class DocumentResponseDTO(
    val id: Long,
    val number: String,
    val validFrom: String,
    val validTo: String,
    val issuer: String,
    val gender: Int,
    val dateOfBirth: String,
    val permanentAddress: String,
    val userId: Long
)